sbi/diagnostics/sbc.py

```diff
@@ -42,45 +46,31 @@ def run_sbc(
       (see https://arxiv.org/abs/2210.04815).
 
     Args:
-        thetas: ground-truth parameters for sbc, simulated from the prior.
-        xs: observed data for sbc, simulated from thetas.
-        posterior: a posterior obtained from sbi. num_posterior_samples: number
-            of approximate posterior samples used for ranking.
+        thetas: Ground-truth parameters for SBC, simulated from the prior.
+        xs: Observed data for SBC, simulated from thetas.
+        posterior: A posterior obtained from sbi.
+        num_posterior_samples: Number of approximate posterior samples used for ranking.
         reduce_fns: Function used to reduce the parameter space into 1D.
             Simulation-based calibration can be recovered by setting this to the
-            string `marginals`. Sample-based expected coverage can be recovered
+            string "marginals". Sample-based expected coverage can be recovered
```

Comment:
I prefer `marginals` or `"marginals"` (both with backticks around them) to get syntax highlighting on the docs.